src/app/cli/src/commands/ui_command.rs

```diff
@@ -64,14 +70,23 @@ impl Command for UICommand {
 
         let web_server = crate::explore::WebUIServer::new(
             self.base_catalog.clone(),
+            self.multi_tenant_workspace,
             self.current_account_name.clone(),
             self.address,
             self.port,
-        );
+        )
+        .await;
 
         let web_server_url = format!("http://{}", web_server.local_addr());
         tracing::info!("HTTP server is listening on: {}", web_server_url);
 
+        if self.get_token {
+            tracing::info!(
+                token = %web_server.get_access_token(),
+                "Issued API server token"
+            );
```

Comment:
I guess we need this for push ingest external client to be able to connect.
But how will it work in real life on the platform? Simply `kamu login` won't be fine for the IoT device case.

Reply:
I think we will need API token issuing feature for IoT device case